gwsumm/data/timeseries.py

```diff
@@ -268,7 +268,7 @@ def _query():
                 break
 
     # validate files existing and return
-    cache = list(filter(os.path.exists, map(_urlpath, cache)))
+    cache = list(filter(os.path.exists, list(map(_urlpath, cache))))
```

Comment:
The internal call to `list()` should be removed:
```suggestion
    cache = list(filter(os.path.exists, map(_urlpath, cache)))
```